CHANGELOG.md

```diff
@@ -43,7 +43,7 @@ be found.
 * LCOW: WORKDIR correct handling [moby/moby#34405](https://github.com/moby/moby/pull/34405)
 * Windows: named pipe mounts [moby/moby#33852](https://github.com/moby/moby/pull/33852)
 
-### Swarm Mode
+### Swarm mode
```

Comment:
Thanks for making the change. I don't want you to have to manually change it each time, hopefully you can automate that?